org.flatpak.galacteek.Galacteek.yaml

```diff
@@ -0,0 +1,167 @@
+app-id: org.flatpak.galacteek.Galacteek
```

Comment:
based on the location of the source code this should be something like `com.github.pinnaculum.galacteek` or `io.github.pinnaculum.galacteek` (either is fine, using flatpak.org is not)